build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -53,301 +53,304 @@ let wrapCounter = 0;
  */
 let transform;
 
-/**
- * Updates the set of preprocessors to run on HTML and JS files before testing.
- * Notes:
- * - The HTML transform is lazy-required because the server is built at startup.
- * - We must use babel on windows until esbuild can natively downconvert to ES5.
- * @param {!RuntimeTestConfig} config
- */
-function updatePreprocessors(config) {
-  const createHtmlTransformer = function () {
-    return function (content, _file, done) {
-      if (!transform) {
-        const outputDir = `../../../${SERVER_TRANSFORM_PATH}/dist/transform`;
-        transform = require(outputDir).transformSync;
-      }
-      done(transform(content));
-    };
-  };
-  createHtmlTransformer.$inject = [];
-  config.plugins.push({
-    'preprocessor:htmlTransformer': ['factory', createHtmlTransformer],
-  });
-  config.preprocessors[karmaHtmlFixturesPath] = ['htmlTransformer', 'html2js'];
-  for (const karmaJsPath of karmaJsPaths) {
-    config.preprocessors[karmaJsPath] = ['esbuild'];
+class RuntimeTestConfig {
```

Comment:
Might be worth adding a comment to indicate that this class consumes the static Karma config and dynamically populates the remaining fields based on the test type and cmd line args.

Reply:
👍